modules/op-release-notes-1-2.adoc

```diff
@@ -21,6 +23,12 @@ In addition to the fixes and stability improvements, here is a highlight of what
 * This release of {pipelines-title} adds support for a disconnected installation.
 //* The `PodSecurityPolicy` configuration was changed to enable {pipelines-title} in a restricted environment by running the tekton pipelines controller and tekton pipelines webhook as non-root users.
 //needs confirmation from Gabe if this needs to be added.
++
+[NOTE]
+====
+Installations in restricted environments are currently not supported on IBM Power Systems, IBM Z, and LinuxONE.
+====
+
```

Comment:
@SNiemann15 I would put this note after line 23. The commented out part on L24 and 25 needs to be deleted.